pom.xml

```diff
@@ -72,6 +72,11 @@ THE SOFTWARE.
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Generally we don't like to add dependencies to Remoting, but this one is small, so it's probably okay.

Reply:
Though do we really need a third-party dependency for exponential backoff? Remoting already retries connection, and typically you need to tune backoff behavior a bit for the caller anyway. E.g. https://github.com/jenkinsci/apache-httpcomponents-client-4-api-plugin/blob/4e7d9a7bae61f816f95f6fc1fb144e6ed40895f1/src/main/java/io/jenkins/plugins/httpclient/RobustHTTPClient.java#L159-L216